databases/week2/part_3.sql

```diff
@@ -0,0 +1,52 @@
+-- Active: 1660903376490@@127.0.0.1@3306@hyf_lession_2
+
+--Part 3: More queries
+
+USE hyf_lession_2;
+
+--Get all the tasks assigned to users whose email ends in @spotify.com
+
+SELECT
+    task.title AS 'Task Name',
+    user.email AS 'email'
+FROM user_task
+    JOIN task ON user_task.task_id = task.id
+    JOIN user ON user_task.user_id = user.id
+WHERE
+    user.email LIKE '%@spotify.com%';
+
```

Comment:
This part is repeated both here and in part 2. Pls remove them from the file part2 or put them into the relevant files.